docs/devices/067761A.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Legrand 067761A control via MQTT"
+description: "Integrate your Legrand 067761A via Zigbee2MQTT with whatever smart home infrastructure you are using without the vendor's bridge or gateway."
+addedAt: 2024-03-31T17:06:22Z
+pageClass: device-page
+---
+
+<!-- !!!! -->
+<!-- ATTENTION: This file is auto-generated through docgen! -->
+<!-- You can only edit the "Notes"-Section between the two comment lines "Notes BEGIN" and "Notes END". -->
+<!-- Do not use h1 or h2 heading within "## Notes"-Section. -->
+<!-- !!!! -->
```

Comment:
Don't know how to generate this based on my local `zigbee-herdsman-converters` and I assume it will be regenerated on build anyway? :shrug: